client/me/purchases/billing-history/utils.jsx

```diff
@@ -62,14 +63,24 @@ export function renderTransactionAmount( transaction, { translate, addingTax = f
 		return transaction.amount;
 	}
 
+	const countryTaxInfo = getVatVendorInfo(
+		transaction.tax_country_code,
+		transaction.date,
+		translate
+	)
+		? getVatVendorInfo( transaction.tax_country_code, transaction.date, translate )
+		: { taxName: 'tax' };
```

Comment:
Actually, this fallback should still be translated. So maybe:

```suggestion
		: { taxName: translate( 'tax', {comment: 'A generic name for taxes in a country, to be used after a tax amount like "$12.34 tax"'} ) };
```

Reply:
Oh! I had a thought. You can avoid having to create a new translation for this fallback by re-using the original translation that you're replacing here! `'(includes %(taxAmount)s tax)'` is already translated, so you can say "if there is no tax name, render the full fallback translation instead of the new one which uses the localized tax name".

Reply:
I think I accomplished this...or some version of it...with my latest [commit](86a662389563e85c0a848a1c04f48b58b7e1f1f8). Not sure if it is a little overcomplicated or not. I was trying to avoid setting up `const taxAmount = addingTax && countryTaxInfo ? [...]  : [...] `. 

Not sure if that was the right approach. Looking forward to your thoughts.